skyportal/handlers/api/followup_request.py

```diff
@@ -1234,6 +1234,65 @@ def delete(self, request_id):
             return self.success()
 
 
+class FollowupRequestCommentHandler(BaseHandler):
+    @permissions(["Upload data"])
+    def put(self, followup_request_id):
+        """
+        ---
+        description: Update a follow-up request comment
+        tags:
+          - followup_requests
+        parameters:
+          - in: path
+            name: followup_request_id
+            required: true
+            schema:
+              type: string
+        responses:
+          200:
+            content:
+              application/json:
+                schema: Success
+          400:
+            content:
+              application/json:
+                schema: Error
+        """
+
+        data = self.get_json()
+        comment = str(data.get("comment")).strip()
+
+        if comment in ["", "None"]:
+            comment = None
+
+        with self.Session() as session:
+            try:
+                stmt = FollowupRequest.select(
+                    session.user_or_token, mode="update"
+                ).where(FollowupRequest.id == followup_request_id)
+                followup_request = session.scalar(stmt)
+
+                if followup_request is None:
+                    return self.error(
+                        f"Followup request {followup_request_id} not found."
+                    )
+
+                followup_request.comment = comment
+                session.commit()
+                self.push_all(
+                    action='skyportal/REFRESH_ALLOCATION_REQUEST_COMMENT',
+                    payload={
+                        'allocation_id': followup_request.allocation_id,
+                        'followupRequest_id': followup_request.id,
+                        'followupRequest_comment': followup_request.comment,
```

Comment:
that's a bit of a weird name: `followupRequest_comment`. You are using both javascript style of variable naming an python's classic snake case. It's not incorrect (I mean the code works), but when designing new features in the future just make sure to use something consistent (even if the current naming scheme is incorrect, you can fix it). For this PR it's fine, feel free to modify it or leave it as is.

Reply:
&#x2611; Done